iib/workers/tasks/opm_operations.py

```diff
@@ -112,13 +114,15 @@ def opm_registry_serve(db_path: str) -> Tuple[int, subprocess.Popen]:
         try:
             cmd = ['opm', 'registry', 'serve', '-p', str(port), '-d', db_path, '-t', '/dev/null']
             cwd = os.path.dirname(db_path)
-            return (
+            result = (
```

Comment:
Minor: Can you share if this change was done to refactor the code or if it was impacting the mypy checks as well?

Reply:
All of the changes in the first commit were done, because of mypy errors. For example in this one, Mypy was expecting some return value after the "possible" end of the loop - which could not happen anyway. 

Should I add some more comments about the changes, or is it okay like this?

Reply:
Oh these comments are more than enough. Thank you :)